R/parseRCode.R

```diff
@@ -8,7 +8,9 @@
 #'
 #' @param code Character string containing R Code to parse
 #' @return An \link{expression} object
-#' @author Romain Francois, Rich Pugh
+#' @author Mike K Smith \email{mstoolkit@@googlemail.com}

```

Comment:
Same as above. We should agree how to handle @author vs @contributor. Please don't change this until we have discussed.

Reply:
Already changed the author's name back.